.travis.yml

```diff
@@ -8,6 +8,9 @@ python:
   - "3.6"
   - "3.7"
 
+env:
+  - CODACY_PROJECT_TOKEN=${CodacyToken}
```

Comment:
From what I'm reading, I don't think this will work - you need to encrypt it https://docs.travis-ci.com/user/environment-variables/#encrypting-environment-variables

Reply:
I am actually following this https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings
I don't see the need to encrypt. In this case the variable is defined under [Settings](https://travis-ci.com/github/urlstechie/urlchecker-python/settings) and is hidden.

Reply:
okay, if that works, then that works!

Reply:
right now it is not, so I am trying to figure the syntax. The alternative you mention would require me to install travis-ci and before that gem and that's extra work I am trying to avoid.

Reply:
I just posted a [question](https://stackoverflow.com/questions/67143625/use-valriable-defined-in-travis-settings-in-automation-script) on it; maybe someone see what is missing.

Reply:
You could spin up a container if you don't want to install things on your host. I think the encrypted way is the right way to do it (at least what I did when I used travis) but I also think you're right that it should work the "easy" way too. I stopped using travis years ago so I don't know anymore.

Reply:
I will give it a shot tomorrow. Actually I found out that the previous Codacy token was useless since we were not using codacy for this project anymore. I am also moving away from Travis since I hit my limit number of runs there.

Reply:
`<personal opinion>` I think the project would be fine without it.

Reply:
I agree, if I cannot fix this, I will drop it. We already have codecov.